jenkins-scripts/dsl/core.dsl

```diff
@@ -31,7 +31,6 @@ update_vcpkg_snapshot_job.with
     }
 }
 
-
```

Comment:
I don't necessarily mind stray whitespace changes but it is worth asking if this was intentional.

Reply:
it was, yes.